omod-1.9/src/test/java/org/openmrs/module/webservices/rest/web/v1_0/controller/openmrs1_9/VisitController1_9Test.java

```diff
@@ -257,6 +257,21 @@ public void shouldVoidAVisit() throws Exception {
 		Assert.assertEquals("test reason", visit.getVoidReason());
 	}
 	
+	@Test
+	public void shouldUnVoidAVisit() throws Exception {
+		Visit visit = service.getVisitByUuid(RestTestConstants1_9.VISIT_UUID);
```

Comment:
Can you try void an existing visit in any of the datasets used for VisitServiceTest in the core platform?

Reply:
I think it's not possible since this setting is in a Web and Module context sensitive environment. Unless I duplicate the datasets into this module.

Reply:
When documenting, avoid exposing internal implementation details to the client.

Reply:
Thank you @dkayiwa , I've taken note of this.